app/views/govuk_publishing_components/components/_textarea.html.erb

```diff
@@ -10,14 +10,15 @@
   error_message ||= nil
   error_items ||= nil
   character_count ||= nil
+  maxlength ||= nil
   hint_id = "hint-#{SecureRandom.hex(4)}"
   has_error ||= error_message || error_items&.any?
   error_id = "error-#{SecureRandom.hex(4)}"
 
-  css_classes = %w(gem-c-textarea govuk-textarea)
+  css_classes = %w(gem-c-textarea__textarea govuk-textarea)
```

Comment:
While I get your point of view regarding this, it introduces inconsistency with the rest of the form components and with the design system class convention. Otherwise the PR looks fine to me.

Reply:
Oh that's strange. Feels a bit wrong for us not to have a root class on a component. 

I'll work a way around that.